THANKS.txt

```diff
@@ -241,6 +241,8 @@ Wesley Alves for improvements to scipy.stats.jarque_bera and scipy.stats.shapiro
 Mark Borgerding for contributing linalg.convolution_matrix.
 Shashaank N for contributions to scipy.signal.
 Frank Torres for fixing a bug with solve_bvp for large problems.
+Romain Jacob for non-paremetric confidence intervals for quantiles
```

Comment:
Can this line be removed from the PR? 
See [thread](https://mail.python.org/pipermail/scipy-dev/2020-August/024331.html) for discussion (changes to this file causes merge conflicts adding effort to contributors and maintainers). 
For now, I've opened a [wiki](https://github.com/scipy/scipy/wiki/*THANKS.txt-additions-modifications-for-1.6.0*)
Once the PR is merged, we add thanks to the wiki. Similar process to what we followed in 1.5.0 release.

Reply:
Done.